documentation/ProjectReference-Protocol.md

```diff
@@ -52,6 +52,7 @@ These targets are all defined in `Microsoft.Common.targets` and are defined in M
   * **New** for MSBuild 15/Visual Studio 2017. Supports the cross-targeting feature allowing a project to have multiple `TargetFrameworks`.
   * **Conditions**: only when metadata `SkipGetTargetFrameworkProperties` for each reference is not true.
   * Skipped for `*.vcxproj` by default.
+  * This target must be present in a build request that does not specify a `TargetFramework`—the “outer” build. It is not required in a fully-specified (“inner”) build.
```

Comment:
I was more concerned about the inverse: the ones below need to be in inner build, but not outer.